README.md

```diff
@@ -93,21 +79,77 @@ versions of these dependencies:
 | gRPC    | v1.10.x |
 | libcurl | 7.47.0  |
 
-For Linux, the `ci/Dockerfile.*` files are a good reference on how to install
-all the necessary dependencies on each distribution. For Windows,
-consult [ci/install-windows.ps1](ci/install-windows.ps1).
+#### Tests
 
-On macOS, the following commands should install all the dependencies you need:
+Integration tests at times use the
+[Google Cloud SDK](https://cloud.google.com/sdk/). The integration tests run
+against the latest version of the SDK on each commit and PR.
+
+## Install Dependencies
+
+#### Fedora
+
+```bash
+dnf makecache
+dnf install autoconf automake c-ares-devel ccache clang clang-tools-extra cmake curl dia doxygen gcc-c++ git golang graphviz  lcov libcurl-devel libtool make ncurses-term openssl-devel pkgconfig python python-gunicorn python-httpbin shtool unzip wget  which zlib-devel
+```
+
+#### Ubuntu (Bionic Beaver)
```

Comment:
I think these instructions should also work for `Xenial Xerus`, which is a LTS release between Trusty and Bionic.  Should we say that?

Reply:
Unless it's certain I'd like to refrain for now.